app/WebParsing/PostParser.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE OverloadedStrings #-}
+module WebParsing.PostParser
+    (parsePosts) where
+
+import Network.HTTP
```

Comment:
Look carefully at what dependencies are actually needed. It's a good practice to start from scratch and just add the dependencies as you need them. (So remove the ones that aren't necessary.)